pkg/model/components/nodeterminationhandler.go

```diff
@@ -51,6 +51,9 @@ func (b *NodeTerminationHandlerOptionsBuilder) BuildOptions(o interface{}) error
 	if nth.EnableRebalanceDraining == nil {
 		nth.EnableRebalanceDraining = fi.Bool(false)
 	}
+	if nth.ExcludeFromLoadBalancers == nil {
+		nth.ExcludeFromLoadBalancers = fi.Bool(false)
```

Comment:
Any downsides to enabling this by default?

Reply:
NTH defaults to false, but in kOps, we can set the default value to true because there is nothing bad effect with it.